doc/tsc-meetings/2015-08-19.md

```diff
@@ -0,0 +1,120 @@
+# Node.js Foundation TSC Meeting 2015-08-19
+
+## Links
+
+* **Audio Recording**: https://soundcloud.com/node-foundation/tsc-meeting-2015-08-19
+* **GitHub Issue**: https://github.com/nodejs/node/issues/2435
+* **Minutes Google Doc**: https://docs.google.com/document/d/1xsj_4UlrLNxahRvC7SpLtFM3D-Ks6CZEqEM5nyj6bjk
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1q2bFjnf0Y23Ljxoze56Pmrbailaj5-UAqIUqIYVhiIk>_
+
+## Agenda
+
+Extracted from **tsc-agenda** labelled issues and pull requests prior to meeting.
+
+* Procedure for rolling out node-accept-pull-request [#2434](https://github.com/nodejs/node/issues/2434)
+* Release procedure changes & nominating @sam-github and @jasnell as releasers [#2416](https://github.com/nodejs/node/issues/2416)
+
+## Minutes
+
+
+### Present
+
+* Mikeal Rogers
+* Rod Vagg (TSC)
+* James Snell (TSC)
+* Michael Dawson (TSC)
+* Steven R Loomis (TSC) 
+* Chris Dickinson (TSC)
+* Alexis Campailla (TSC)
+* Brian White (TSC)
+* Jeremiah Senkpiel (TSC)
+* Shigeki Ohtsu (TSC)
+* Trevor Norris (TSC)
+* Domenic Denicola
+* Ben Noordhuis (TSC)
+* Colin Ihrig (TSC)
```

Comment:
![screen shot 2015-08-21 at 8 08 45 pm](https://cloud.githubusercontent.com/assets/495647/9406062/7a734922-4840-11e5-83a6-39cc6b54b372.png)

yes, just quiet, like a mouse that you didn't notice